src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -123,6 +132,12 @@
             this::delete,
             "Takes the specified parameters and calls CatalogFramework::query. `params` takes:"
                 + " `ids` (Required, value: List(String))"));
+
+    builder.put(
+        "ddf.catalog/clone",
```

Comment:
:exclamation: The goal of the CatalogMethods JSON RPC API was to mirror the Catalog Framework API as close as possible.  I think this clone method belongs in a GraphQL data resolver or in a new JSON RPC endpoint instead.